backend/src/crd/samples/viewer/tensorboard-local-volume.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This example shows how to use a Kubernetes local persistent volume with Tensorboard
+# viewer. The definition below assumes you have a volume called
+# 'argo-volume' in the cluster. This persistent volume will be mounted to
```

Comment:
should be "have a volume called '**argo-claim**' in the cluster" in the below definition?

Reply:
I believe it's correct, the volume should be called `argo-volume`. The PVC name is `argo-claim`.